examples/tutorials/01_The_Basic_Tools_of_the_Deep_Life_Sciences.ipynb

```diff
@@ -790,31 +827,50 @@
       "source": [
         "## Converting NumpyDataset to `tf.data`\n",
         "\n",
-        "This can be easily done by the `make_iterator()` method of `NumpyDataset`. This converts the `NumpyDataset` to `tf.data`. Let's look how it's done!"
+        "This can be easily done by the `itersamples()` method of `NumpyDataset`. This converts the `NumpyDataset` to `tf.data`. Let's look how it's done!"
       ]
     },
     {
       "cell_type": "code",
       "metadata": {
         "id": "hVy39LEe5zJA",
         "colab_type": "code",
-        "colab": {}
+        "colab": {
+          "base_uri": "https://localhost:8080/",
+          "height": 102
+        },
+        "outputId": "62a7d6d9-7c1d-4ba1-f1a7-9a83249e9829"
       },
       "source": [
-        "# iterator_ = dataset_.make_iterator() # Using make_iterator for converting NumpyDataset to tf.data\n",
-        "# next_element_ = iterator_.get_next()\n",
+        "dataset = np.zeros((4,5))\n",
+        "labels = np.zeros((4,))\n",
         "\n",
-        "# sess = tf.Session() # tensorflow session \n",
-        "# data_and_labels = sess.run(next_element_) # data_ contains the data and label_ contains the labels that we fed in the previous step\n",
+        "counter = 0\n",
+        "for x, y, w, ids in dataset_.itersamples():\n",
```

Comment:
I think you can do this much simpler as 
```
tf_dataset = tf.data.Dataset.from_tensor_slices((dataset_.X, dataset_.y))
```
by using the `X`, `y` attributes of `Dataset` objects

Reply:
Thanks!